xalia/Win32/NonclientScrollProvider.cs

```diff
@@ -430,7 +430,7 @@ public async Task<double> GetMinimumIncrementAsync(UiDomElement element)
                         throw new Win32Exception();
 
                     return CalculateMinimumIncrement(sbi, si);
-                }, Tid);
+                }, CommandThreadPriority.Query);

```

Comment:
And here. (I'm beginning to guess these were just incorrectly marked as `CommandThreadPriority.User` to begin with and fixed now)